### PR TITLE
Fix 3.0.0 bugs #fixed

### PR DESCRIPTION
**Pull request by Kaspik**
## Changes:
- Fix time formatter not accepting 24hour format
- Fix missing xib files for non EN users
- Fix query comment out shortcut

## Closes following issues:
- Closes https://github.com/Sequel-Ace/Sequel-Ace/issues/651
- Closes https://github.com/Sequel-Ace/Sequel-Ace/issues/649
- Closes https://github.com/Sequel-Ace/Sequel-Ace/issues/639

## Tested:
- Processors:
  - [ ] Intel
  - [x] Apple Silicon
- macOS versions:
  - [ ] 10.12
  - [ ] 10.13
  - [ ] 10.14
  - [ ] 10.15
  - [x] 11.1
- Xcode version: 12.3

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Please use one of these hashtags for your PR title:
   - [ ] 10.13
   - [ ] 10.14
   - [ ] 10.15
-  - [ ] 11.0
+  - [ ] 11.1
```
Comment (Jason-Morcos) [inline .github/pull_request_template.md:27]: Probably should just make this 11, as Apple now appears to be using all three build numbers unlike they really only used the last 2

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ about: Create a report to help us improve
 - macOS Version:
 - Processor Type (Intel/Apple):
 - MySQL Version:
+- macOS localization:
```
Comment (Jason-Morcos) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Maybe capitalize "localization"?

File: .github/pull_request_template.md (hunk 1 above)
Comment (Jason-Morcos) [reply .github/pull_request_template.md:27]: Could also add localization checkboxes for the testing things so we make sure things are tested in various localizations?

File: Source/Other/Extensions/DateFormatterExtension.swift [hunk 3]
```diff
@@ -41,7 +41,7 @@ extension DateFormatter {
 	/// 2020-06-30 14:14:11 is  example
 	@objc public static var naturalLanguageFormatter: DateFormatter = {
 		let formatter = DateFormatter()
-		formatter.dateFormat = "yyyy-MM-dd hh:mm:ss"
+		formatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
```
Comment (Jason-Morcos) [inline Source/Other/Extensions/DateFormatterExtension.swift:44]: Shouldn't this probably be dependent on the user's localization settings? If it really is natural language, some parts of the world are 24hr time and others aren't

Comment (Kaspik) [reply Source/Other/Extensions/DateFormatterExtension.swift:44]: nah, `hh` is 12 hours only, `HH` is both 12 and 24 hour ... that was my mistake, so it returned value only until noon 😂

Comment (Jason-Morcos) [reply Source/Other/Extensions/DateFormatterExtension.swift:44]: Right, but in America you would want 12 hour time suffixed by AM/PM lol. Although most devs should be comfy with 24hr time

Comment (Kaspik) [reply Source/Other/Extensions/DateFormatterExtension.swift:44]: I don't think mysql can return that - this is for parsing the mysql format

**Pull request merged**